styles.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html body {
+  min-width: 375px;
+  height: 100%;
+  font-family: 'Poppins', sans-serif;
+  margin-left: 3rem;
+  margin-right: 3rem;
+}
+
+.navigation {
+  background-color: white;
+  padding-top: 1rem;
+  padding-right: 3rem;
+  padding-left: 3rem;
+  width: 100%;
+  height: 20px;
+  display: flex;
+  font-size: 18px;
+  justify-content: space-between;
+}
+
+.toggle {
+  padding-right: 3rem;
+}
+
+.menu {
+  border: none;
+  color: #6070ff;
+}
+
+a {
+  text-decoration: none;
+  font-style: normal;
+  font-family: 'poppins', sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+a:hover {
+  color: green;
+}
+
+.first-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 4rem;
+  padding-right: 3rem;
+}
```

Comment:
Kindly adjust the `padding-top` and `padding-right` to match the Figma design.

Reply:
I have adjusted the margin-left and margin-right to fit the Figma design.
I have removed the height and adjusted the padding to match the design.
Also, I have adjusted the padding-top and padding-right to match the design.